test/utils/TestConstants.sol

```diff
@@ -1,4 +1,4 @@
 // SPDX-License-Identifier: UNLICENSED
 pragma solidity ^0.8.25;
 
-uint32 constant TEST_DEFAULT_VALIDATION_ENTITY_ID = 1;
+uint32 constant TEST_DEFAULT_VALIDATION_ENTITY_ID = type(uint32).max;
```

Comment:
This will cause the default validation to coincide with the direct call validation entity ID. Could you double-check that this doesn't break assumptions for those tests?

Reply:
So I dug into this a good bit, the only time we actually care about the entityId for direct-call validations is to build the entire ModuleEntity-- so the last 4 bytes will be 0xff in both cases, but since we only index with the full ModuleEntity, we're reliant on the module address anyway.